imports/plugins/included/marketplace/register.js

```diff
@@ -0,0 +1,41 @@
+import { Reaction } from "/server/api";
+
+Reaction.registerPackage({
+  label: "Marketplace",
+  name: "reaction-marketplace",
+  icon: "fa fa-globe",
+  autoEnable: true,
+  settings: {
+    name: "Marketplace",
+    public: {
+      allowGuestSellers: true
+    }
+  },
+  registry: [
+    // page
+
+    // Dashboard card
+    {
```

Comment:
I wonder if we should just place the settings under the `Catalog` card, and not give Marketplace it's own dashboard card (we're trying to consolidate these as much as possible).  All the registry entries would still be here.  Easy to move later one way or another though.